programs/lib/include/flexpret_io.h

```diff
@@ -20,8 +20,9 @@ static inline void write_tohost(uint32_t val) {
   write_tohost_tid(tid, val);
 }
 
-#define _fp_abort(reason) do { \
-  printf("%s: %i: " PRINTF_COLOR_RED "Abort: %s\n" PRINTF_COLOR_NONE, __FILE__, __LINE__, reason); \
+#define _fp_abort(fmt, ...) do { \
+  printf("%s: %s: %i: " PRINTF_COLOR_RED "Abort:" PRINTF_COLOR_NONE, __FILE__, __func__, __LINE__); \
+  printf(fmt, ##__VA_ARGS__); \
   write_tohost(CSR_TOHOST_ABORT); \
 } while(0)
```

Comment:
Same as with fp_assert(). This macro should probably not be prefixed with an underscore, however.